src/test/java/org/folio/marccat/resources/AuthorityRecordTest.java

```diff
@@ -10,46 +10,53 @@
 import org.folio.marccat.StorageTestSuite;
 import org.folio.marccat.TestBase;
 import org.folio.marccat.resources.domain.FixedField;
+import org.junit.FixMethodOrder;
 import org.junit.Test;
 import org.junit.runner.RunWith;
+import org.junit.runners.MethodSorters;
 import org.springframework.test.context.ActiveProfiles;
 import org.springframework.test.context.junit4.SpringRunner;
 
 import com.fasterxml.jackson.databind.ObjectMapper;
 
+import io.restassured.response.Response;
+
 /**
  * @author elena
  *
  */
 @RunWith(SpringRunner.class)
 @ActiveProfiles("test")
+@FixMethodOrder(MethodSorters.NAME_ASCENDING)
 public class AuthorityRecordTest extends TestBase {
 
   private static final String AUTHORITY_RECORD_URL = "/marccat/authority-record";
   private static final String CONTENT_TYPE = "Content-Type";
   private static final String FILE_TYPE = "application/json";
+  private static String authorityId;
 
   @Test
-  public void saveNameReturn201Status() throws IOException {
+  public void test1SaveReturn201Status() throws IOException {
     String url = getURI(AUTHORITY_RECORD_URL);
     Map<String, String> headers = addDefaultHeaders(url, StorageTestSuite.TENANT_ID);
     String templateJson = getTemplateJson("/authority/name.json");
 
-    given()
+    Response myResponse = given()
       .headers(CONTENT_TYPE, FILE_TYPE)
       .headers(headers)
       .queryParam("view", "-1")
       .queryParam("lang", "eng")
       .body(templateJson)
       .when()
-      .post(url)
-      .then()
-      .statusCode(201);
+        .post(url);
+    authorityId = myResponse.jsonPath().get("body").toString();
```

Comment:
Some tests should be done on the authorityId ?

Reply:
authorityId is used to test the api GET